fuse/file.go

```diff
@@ -168,6 +180,16 @@ func (fi *File) Listxattr(ctx context.Context, req *fuse.ListxattrRequest, resp
 	return nil
 }
 
+// Readlink reads a symbolic link.
+// This call is triggered when OS tries to see where symlink points
+func (fi *File) Readlink(ctx context.Context, req *fuse.ReadlinkRequest) (string, error) {
+	info, err := fi.m.fs.Stat(fi.path)
+	if err != nil {
+		return "/brig/backend/ipfs/", err
+	}
+	return fmt.Sprintf("/brig/backend/ipfs/%s", info.BackendHash), nil
```

Comment:
Hmm, not so sure about the intention of this feature. What would the user supposedly do with the link in here?
Programs that call ``readlink()`` will detect this as ``ENOENT`` and just show a broken link. The text above would be only seen if somebody calls the ``readlink`` utility on the file.

Reply:
If we do not implement `Readlink` then tools like `ls` would spam the output with
```
ls: cannot read symbolic link '/tmp/zz/README.md': Input/output error
```
Where do we point a link to noncached file is good question. I chose to point it quite wordy link which somewhat describe the issue. This way
a user can get some hints without calling `brig` (if someone is using fuse, they probably are shielded from `brig`). Also one can do a shell only script to ask `ipfs` to download missing hash.

Reply:
Okay. I guess that's good enough for now. Maybe later we could do some more tricks with the link content.